omnipaxos/tests/atomic_storage_test.rs

```diff
@@ -526,60 +525,3 @@ fn atomic_storage_majority_promises_test() {
         run_single_test(i);
     }
 }
-
-#[test]
-#[serial]
-fn atomic_storage_majority_accepted_stopsign_test() {
```

Comment:
These tests are not needed anymore?

Reply:
The purpose of the test was to check for the atomicity of updating `decided_idx` and the result of the `ss.decided(idx)` function. It was a strange test because if you need to roll back the `decided_idx` you dont have to rollback anything about the stopsign (even when it included the `log_idx` field). But I mainly removed it because the `ss.decided(idx)` function no longer exists.